recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-core/images/core-image-thin-initramfs.bbappend

```diff
@@ -29,17 +31,57 @@ python __anonymous () {
         d.appendVar("IMAGE_INSTALL", " domu domu-run domu-install-artifacts")
 }
 
-generate_uboot_image() {
-    ${STAGING_BINDIR_NATIVE}/uboot-mkimage -A arm64 -O linux -T ramdisk -C gzip -n "uInitramfs" \
-        -d ${DEPLOYDIR}-image-complete/${IMAGE_LINK_NAME}.cpio.gz ${DEPLOY_DIR_IMAGE}/${IMAGE_NAME}${IMAGE_NAME_SUFFIX}.cpio.gz.uInitramfs
-    ln -sfr  ${DEPLOY_DIR_IMAGE}/${IMAGE_NAME}${IMAGE_NAME_SUFFIX}.cpio.gz.uInitramfs ${DEPLOY_DIR_IMAGE}/uInitramfs
+IMAGE_ROOTFS_SIZE = "65535"
+
+WKS_FILE = "prod-devel.wks"
+IMAGE_FSTYPES += "wic.xz wic.bmap"
```

Comment:
please check that bmap tool supports compressed images

Reply:
> please check that bmap tool supports compressed images

@andr2000 my understanding- yes
"
_Raw images compress extremely well because the holes are essentially zeroes,
which compress perfectly. This is why 3.7GiB Tizen IVI raw images, which
contain about 1.1GiB of mapped blocks, take only 300MiB in a compressed form.
And the important point is that you  need to decompress them only while
flashing. The bmaptool does this "on-the-fly"._
"

from https://github.com/intel/bmap-tools/blob/master/docs/README

Reply:
http://manpages.ubuntu.com/manpages/xenial/man1/bmaptool.1.html
IMAGE may be compressed,  in  which  case  bmaptool  decompresses  it  on-the-fly.   The
         compression  type  is  detected  by  the file extension and the following extensions are
         supported:

             1. ".gz", ".gzip", ".tar.gz" and ".tgz" for files and tar archives  compressed  with
             "gzip" program
             2.  ".bz2",  "tar.bz2",  ".tbz2",  ".tbz",  and  ".tb2"  for  files and tar archives
             compressed with "bzip2" program
             3. ".xz", ".tar.xz", ".txz" for files and tar archives compressed with "xz" program
             4. ".lzo", "tar.lzo", ".tzo" for  files  and  tar  archives  compressed  with  "lzo"
             program